.travis.yml

```diff
@@ -59,7 +56,9 @@ jobs:
       env:
         - STACK_VERSION=7.0.0-alpha1-SNAPSHOT
         - SCOPE=@elastic/apm-rum
-
+    - stage: test
```

Comment:
any reason why we are running docs as part of test? It's better to keep it as separate stage IMO.

Reply:
Travis won't start the next stage until the previous stage is finished. With our concurrency level having two stages improves overall speed.

Reply:
Got it. makes total sense to do this.